addons/mail/static/tests/discuss/core/web/chat_window_new_message.test.js

```diff
@@ -152,3 +152,31 @@ test('open chat from "new message" should open chat window', async () => {
     await contains(".o-mail-ChatWindow", { count: 0, text: "New message" });
     await contains(".o-mail-Thread");
 });
+
+test("Opening 'new message' twice keeps new message chat window in place", async () => {
```

Comment:
Should it stay in place? Don't we want to make it the first one?

Reply:
TBH both work fine, the exact positioning doesn't really matter.

The test is just a pretext for coverage of clicking on "new message" twice